Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM openjdk:8-jdk-alpine
+FROM openjdk:8-jre-slim
 
-MAINTAINER Johannes Molland <johannes.molland@difi.no>
+RUN groupadd -o -g 1000 java \
+    && useradd -o -r -m -u 1000 -g 1000 java
 
-ARG jarPath
-ADD ${jarPath} app.jar
+ENV APP_DIR=/opt/digdir \
+    JAVA_OPTS=""
 
-ENV JAVA_OPTS=""
+ADD /target/*.jar ${APP_DIR}/app.jar
 
-ENTRYPOINT [ "sh", "-c", "exec java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
\ No newline at end of file
+RUN chown -R java:java ${APP_DIR}
+RUN chmod +x ${APP_DIR}/*
+
+USER java
+
+ENTRYPOINT [ "sh", "-c", "exec java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar ${APP_DIR}/app.jar" ]
```

Comment:
La ikkje merke til dette sist, men -Djava.security...-innstillinga trengs ikkje lenger. Sjekk med Google, men dette behovet er fjerna for nokre år sidan.

Reply:
Ser du har rett: https://stackoverflow.com/questions/59564269/deprecated-java-security-egd-file-dev-urandom-for-spring-boot-applications

Da fjerner jeg den.